test/parallel/test-tls-multi-key.js

```diff
@@ -2,52 +2,162 @@
 const common = require('../common');
 const assert = require('assert');
 
+// Test multi-identity ('key')/multi-algorithm scenarios.
+
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
 const tls = require('tls');
 const fs = require('fs');
 
-const options = {
+// Key and cert are ordered as ec, rsa.
```

Comment:
I understand it was the same as the previous. I think this PR is a good time to change this. It is not common situation to have two multi certs that one is self-cert and the other is chained with different cert type(ECC and RSA). And in openssl, x509_verify procedure has different code path in case of self-signed. 
This test intends to check two different cert types of ECC and RSA in the same cert store so that I think it is better to have the same cert chain structure tier for checking only two algorithms.